vendor/cran.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+const Joi = require('joi');
+const ServiceTester = require('./runner/service-tester');
+
+const t = new ServiceTester({ id: 'cran', title: 'CRAN/METACRAN' });
+module.exports = t;
+
+t.create('version')
+  .get('/v/devtools.json')
+  .expectJSONTypes(Joi.object().keys({
+    name: Joi.equal('cran'),
+    value: Joi.string().regex(/^v\d+\.\d+\.\d+$/)
+  }));
+
+t.create('specified license')
+  .get('/l/devtools.json')
+  .expectJSON({ name: 'license', value: 'GPL (>= 2)' });
+
+t.create('unknown package')
+  .get('/l/some-bogus-package.json')
```

Comment:
Is it a good idea to send invalid requests to a production API? Might be worth mocking this with a 404 response rather than sending a real request 😛